docs/RELEASE_NOTES.md

```diff
@@ -63,6 +63,13 @@
 empty optionals in a supplied list (with a varargs variation) returning a list of the contents of the non-empty 
 optionals. [#255](https://github.com/commercetools/commercetools-sync-java/issues/255) 
 
+**Breaking Changes** (6)
+- **ProductType Sync** - Renamed `ProductTypeUpdateAttributeDefinitionActionUtils` to `AttributeDefinitionsUpdateActionUtils`. [#302](https://github.com/commercetools/commercetools-sync-java/issues/302)
+- **ProductType Sync** - Renamed `ProductTypeUpdateLocalizedEnumActionUtils` to `LocalizedEnumsUpdateActionUtils.` [#302](https://github.com/commercetools/commercetools-sync-java/issues/302)
+- **ProductType Sync** - Renamed `ProductTypeUpdatePlainEnumActionUtils` to `PlainEnumsUpdateActionUtils`. [#302](https://github.com/commercetools/commercetools-sync-java/issues/302)
+- **ProductType Sync** - Renamed `LocalizedEnumUpdateActionsUtils` to `LocalizedEnumUpdateActionUtils`. [#302](https://github.com/commercetools/commercetools-sync-java/issues/302)
+- **ProductType Sync** - Renamed `PlainEnumUpdateActionsUtils` to `PlainEnumUpdateActionUtils`. [#302](https://github.com/commercetools/commercetools-sync-java/issues/302)
+- **ProductType Sync** - Renamed `EnumUpdateActionsUtils` to `EnumsUpdateActionUtils`. [#302](https://github.com/commercetools/commercetools-sync-java/issues/302)
```

Comment:
But it wasn't `EnumUpdateActionsUtils`. It was `ProductTypeUpdateEnumActionsUtils` 😉

Reply:
Yes, sir. 
https://github.com/commercetools/commercetools-sync-java/pull/316/commits/20033760604f8378b2c8dd7d2ff38c5513050417